nixui/nix/lib.nix

```diff
@@ -1,6 +1,6 @@
 let
   inherit (import <nixpkgs> {}) pkgs lib;
-  inherit (import <nixpkgs/nixos> {}) config;
+  inherit (import <nixos> {}) config;
   modulesPath = <nixpkgs/nixos/modules>;
```

Comment:
```
modulesPath = <nixos/modules>;
```

Reply:
resolved by https://github.com/nix-gui/nix-gui/pull/210/commits/3f32e25750fd855457afe68f8e9b2228229c66a1

Reply:
lets hope this works ; )

Reply:
All tests pass and smoke test looks fine.